doc/api/zlib.md

```diff
@@ -396,60 +399,61 @@ Provides an object enumerating Zlib-related constants.
 added: v0.5.8
 -->
 
-Returns a new [Deflate][] object with an [options][].
+Passes [`options`][] to the `Deflate` constructor and returns a new [`Deflate`][] object.
 
 ## zlib.createDeflateRaw([options])
 <!-- YAML
 added: v0.5.8
 -->
 
-Returns a new [DeflateRaw][] object with an [options][].
+Passes [`options`][] to the `DeflateRaw` constructor and returns a new [`DeflateRaw`][] object.
 
 ## zlib.createGunzip([options])
 <!-- YAML
 added: v0.5.8
 -->
 
-Returns a new [Gunzip][] object with an [options][].
+Passes [`options`][] to the `Gunzip` constructor and returns a new [`Gunzip`][] object.
 
 ## zlib.createGzip([options])
 <!-- YAML
 added: v0.5.8
 -->
 
-Returns a new [Gzip][] object with an [options][].
+Passes [`options`][] to the `Gzip` constructor and returns a new [`Gzip`][] object.
 
 ## zlib.createInflate([options])
 <!-- YAML
 added: v0.5.8
 -->
 
-Returns a new [Inflate][] object with an [options][].
+Passes [`options`][] to the `Inflate` constructor and returns a new [`Inflate`][] object.
 
 ## zlib.createInflateRaw([options])
 <!-- YAML
 added: v0.5.8
 -->
 
-Returns a new [InflateRaw][] object with an [options][].
+Passes [`options`][] to the `InflateRaw` constructor and returns a new [`InflateRaw`][] object.
 
 ## zlib.createUnzip([options])
 <!-- YAML
 added: v0.5.8
 -->
 
-Returns a new [Unzip][] object with an [options][].
+Passes [`options`][] to the `Unzip` constructor and returns a new [`Unzip`][] object.
 
 ## Convenience Methods
 
 <!--type=misc-->
 
-All of these take a [Buffer][] or string as the first argument, an optional
-second argument to supply options to the `zlib` classes and will call the
-supplied callback with `callback(error, result)`.
+All of these take a [Buffer][] or string as the first argument and an optional 
+second argument to supply options to the `zlib` classes.
+
+All the methods have asynchronous and synchronous forms. The asynchronous 
+form always takes a completion callback as its last argument in
+the form `callback(error, result)`.
 
-Every method has a `*Sync` counterpart, which accept the same arguments, but
-without a callback.
```

Comment:
I would prefer this not be removed without the above additions explaining the distinction that the `*Sync` label is what indicates a method is synchronous. Your current wording could be misinterpreted as meaning that the exact same method could become synchronous by simply not giving it a callback.